deployment/docker/bundle/README.md

```diff
@@ -1,14 +1,31 @@
+<<<<<<< HEAD
```

Comment:
there are conflicts and please update the PR to include only the translation Changes